Makefile

```diff
@@ -182,6 +184,7 @@ TEST_DIRS+=github.com/vmware/vic/cmd/rpctool
 TEST_DIRS+=github.com/vmware/vic/portlayer
 TEST_DIRS+=github.com/vmware/vic/pkg
 TEST_DIRS+=github.com/vmware/vic/apiservers/portlayer
+TEST_DIRS+=github.com/vmware/vic/machines/vic
```

Comment:
machines is intended for VM definitions - more infrastructure than code.

Go repo convention appears to be that `main` packages go under a top level `cmd/` directory.
I'd put the non-main components under `install/` (at least for now that's what they're used for).